### PR TITLE
Fixed deprecated function

### PR DESCRIPTION
**Pull request by Mattio-cmd**
This change should make `:Lspsaga diagnostic()` work properly

### CODE REVIEW

File: lua/lspsaga/diagnostic.lua [hunk 1]
```diff
@@ -32,7 +32,7 @@ function M.lsp_jump_diagnostic_next(opts)
   return _iter_diagnostic_move_pos(
     "DiagnosticNext",
     opts,
-    vim.lsp.diagnostic.get_next_pos(opts)
```
Comment (Mattio-cmd) [inline lua/lspsaga/diagnostic.lua:35]: This is deprecated and should not be used

File: lua/lspsaga/diagnostic.lua [hunk 2]
```diff
@@ -32,7 +32,7 @@ function M.lsp_jump_diagnostic_next(opts)
...
+    vim.diagnostic.get(opts)
```
Comment (Mattio-cmd) [inline lua/lspsaga/diagnostic.lua:35]: As nvim suggest, if you run `:h deprecated` you can see this is the correct way

**Comment by tami5** [comment]
fixed in https://github.com/tami5/lspsaga.nvim

**Comment by Mattio-cmd** [comment]
> fixed in https://github.com/tami5/lspsaga.nvim

Yes, im now using your fork instead of this one. Thanks :)

**Pull request merged**